features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx

```diff
@@ -169,21 +175,32 @@ function SampleAPI(prop) {
                     />
                 </Typography>
                 <div className={classes.actions}>
+                    <APIProductCreateMenu buttonProps={{
+                        size: 'small',
+                        color: 'primary',
+                        variant: 'contained',
+                        className: classes.buttonLeft,
+                    }}
+                    >
+                        <Create />
+                        <FormattedMessage id='Create.New.Api.Product' defaultMessage='Create New API Product' />
```

Comment:
Use the proper format of id. 

The Id should be `Apis.Listing.SampleAPI.SampleAPIProduct.create.new.api.product`

Reply:
<Relative_path_of_the_file>.message.content

Reply:
https://github.com/wso2/carbon-apimgt/pull/8279/commits/898e7283d2ef894f03b48909bef6d76165a42dc8